src/frontend/apps/impress/src/features/docs/docs-grid/components/DocsGrid.tsx

```diff
@@ -134,22 +136,27 @@ export const DocsGrid = ({
                   <DocGridContentList docs={docs} />
                 )}
               </Box>
+              {hasNextPage && !loading && (
+                <InView
+                  data-testid="infinite-scroll-trigger"
+                  as="div"
+                  onChange={loadMore}
+                  rootMargin="200px 0px"
+                  root={scrollRef.current ?? undefined}
```

Comment:
I don't have the feeling that you need `scrollRef`. By putting back `InView` inside the main Box it will work automatically.

```suggestion
```